generators/server/templates/src/main/java/package/web/rest/AccountResource.java.ejs

```diff
@@ -23,7 +23,7 @@ import <%=packageName%>.service.UserService;
 import <%=packageName%>.service.dto.UserDTO;
 import <%=packageName%>.web.rest.errors.InternalServerErrorException;
 
-import com.codahale.metrics.annotation.Timed;
+import io.micrometer.core.annotation.Timed;
```

Comment:
I think the imported can be removed (as the methods are not annotated with `@Timed` anymore.